types/index.d.ts

```diff
@@ -1140,6 +1140,12 @@ declare namespace RamdaAdjunct {
         argsPass<T>(combiningPredicate: (fn: (a: T) => boolean) => (list: T[]) => boolean, predicates: Pred[]): Pred;
         argsPass<T>(combiningPredicate: (fn: (a: T) => boolean) => (list: T[]) => boolean): (predicates: Pred[]) => Pred;
 
+        /**
+         * Takes a function as an argument and a number of additional arguments, and returns a function that just calls the original function given.
```

Comment:
Please align the description of the function with the one defined in https://github.com/char0n/ramda-adjunct/pull/1321/files#diff-00827205c5757eed238b3d5634928ae5R8